site/src/gatsby-theme-marketing-sanity/components/serializers.js

```diff
@@ -16,7 +17,17 @@ export default {
     button: Button,
     callout: Callout,
     code: CodeBlock,
-    'page-image': Figure,
+    'page-image': ({ node }) => (
```

Comment:
This feels pretty rough — we can merge this, but let's come back to it and make this better. Opened #13 to track this.